src/ekat/util/ekat_string_utils.hpp

```diff
@@ -37,6 +37,21 @@ std::string strint (const std::string& s, const int i);
 // Conver the string to all upper case
 std::string upper_case (const std::string& s);
 
+// Computing similarity index between s1 and s2 using Jaro algorithm
```

Comment:
Can you describe briefly some typical output from this algorithm and/or give a reference to it above the function declaration? That will help folks determine where it's needed. And if you have a typical use case in mind, might want to include that here too!